ERRORCODES.md

```diff
@@ -0,0 +1,73 @@
+# Interaction API error codes
+
+## Client Errors
+| Error Code | Description |
+|--|--|
+| *10001* | The Dev name of the Conversation Channel Definition in the request isn't valid for the provided value in the *"to":"\<channelAddressIdentifier\>"* field. |
+| *10002* | The channel type for the provided *"to":"\<channelAddressIdentifier\>"* field isn't valid. |
+| *10003* | The supplied Authorization access token is missing the required Connected App scope. Required value: interaction_api. |
+| *10004* | The supplied Authorization access token is not valid. |
+| *10005* | The AuthorizationContext for the request isn't valid based on the supplied Authorization access token. |
+| *10006* | The channel provider for the provided *"to":"\<channelAddressIdentifier\>"* field isn't valid |
+| *10007* | The file ID field is empty|
+| *10008* | The file size exceeds the maximum limit of 25 MB|
+| *10009* | The file is empty or doesn’t exist|
+| *10010* | The interaction contentLength field is empty|
+| *10011* | The number of interactions of type AttachmentInteraction does not match the number of attachment files|
+| *10012* | Enter a valid workItemId for the MessagingSession ID.|
+| *10013* | Enter either a capacityPercentage or a capacityWeight, but not both.|
+| *10014* | Enter a capacity weight greater than or equal to 0.|
+| *10015* | Enter a capacity percentage between 0 and 100, inclusive.|
+| *10016* | Enter a valid header.|
```

Comment:
The header is too generic here, we have a couple of attributes in header, can we narrow down here?

Reply:
@james-wang-sfdc we use the same error code for all header related validation.. we would have to make changes in our code base prior to making changes to the description. I will leave this as is for now and once we update the code, we can update back here.